tests/contrib/kafka/test_kafka.py

```diff
@@ -16,6 +16,8 @@
 from ddtrace.contrib.kafka.patch import unpatch
```

Comment:
We should add a [subprocess](https://github.com/DataDog/dd-trace-py/blob/c6808b2d6d016aa600b7c52c50b172eeca44bd0d/tests/contrib/redis/test_redis_cluster_asyncio.py#L342) test to ensure the dsm atexit hooks generate the expected traces/payloads.